### PR TITLE
[hironx py] add error msg when checkEncoders did not go well

### PR DESCRIPTION
**Pull request by 130s**

### CODE REVIEW

**Comment by 130s** [comment]
This change prints error message when either one of the following methods (`rh_svc` is hrpsys [RobotHardware component](http://fkanehiro.github.io/hrpsys-base/db/d5b/interfaceOpenHRP_1_1RobotHardwareService.html)) returned false { rh_svc.initializeJointAngle, rh_svc.power, rh_svc.servo }, during the run of `checkEncoders`.

Currently `checkEncoders` returns void, which I wanted to change as well. I won't change it because I'm afraid that doing so breaks API.

File: hironx_ros_bridge/src/hironx_ros_bridge/hironx_client.py [hunk 1]
```diff
@@ -667,13 +667,20 @@ def checkEncoders(self, jname='all', option=''):
...
         self.goActual()  # This needs to happen before turning servo on.
         time.sleep(0.1)
-        self.rh_svc.servo(jname, SWITCH_ON)
+        is_result_hw = is_result_hw and self.rh_svc.servo(jname, SWITCH_ON)
+        if not is_result_hw:
+            # The step described in the following msg is confirmed by the manufacturer 12/14/2015
+            print('Turning servos ({}) failed. This is likely because of issues ' +
+                  "happening in lower level. Please check if the Kawada's " +
+                  "proprietary tool NextageOpenSupervisor returns without issue " +
+                  "or not. If the issue persists, contact the manufacturer.".format(jname))
```
Comment (130s) [inline hironx_ros_bridge/src/hironx_ros_bridge/hironx_client.py:683]: `format(jname)` here doesn't get printed as intended.

**Comment by 130s** [comment]
This PR is suspicious for causing https://github.com/tork-a/rtmros_nextage/issues/227.

File: hironx_ros_bridge/src/hironx_ros_bridge/hironx_client.py [hunk 2]
```diff
@@ -667,13 +667,20 @@ def checkEncoders(self, jname='all', option=''):
...
         print self.configurator_name, 'calib-joint ' + jname + ' ' + option
-        self.rh_svc.initializeJointAngle(jname, option)
+        is_result_hw = is_result_hw and self.rh_svc.initializeJointAngle(jname, option)
         print self.configurator_name, 'done'
-        self.rh_svc.power('all', SWITCH_OFF)
+        is_result_hw = is_result_hw and self.rh_svc.power('all', SWITCH_OFF)
         self.goActual()  # This needs to happen before turning servo on.
         time.sleep(0.1)
-        self.rh_svc.servo(jname, SWITCH_ON)
+        is_result_hw = is_result_hw and self.rh_svc.servo(jname, SWITCH_ON)
```
Comment (130s) [inline hironx_ros_bridge/src/hironx_ros_bridge/hironx_client.py:677]: I confirmed that reverting this single line resolves the issue https://github.com/tork-a/rtmros_nextage/issues/227 but I'm not sure why.
@emijah Could you take a look at this?

Comment (emijah) [inline hironx_ros_bridge/src/hironx_ros_bridge/hironx_client.py:677]: @130s You are assigning (is_result_hw and self.rh_svc.initializeJointAngles(jname, option)) to is_result_w in line 672. Is this what you want?

Comment (130s) [inline hironx_ros_bridge/src/hironx_ros_bridge/hironx_client.py:677]: Ah, [`RobotHardwareService_impl::initializeJointAngle`](https://github.com/fkanehiro/hrpsys-base/blob/eb09ae7d6345435706f11bbb642fe018e0badd1b/rtc/RobotHardware/RobotHardwareService_impl.cpp#L119) returning `void`, not `bool`...

Comment (emijah) [inline hironx_ros_bridge/src/hironx_ros_bridge/hironx_client.py:677]: Oh, you're keeping tabs

Comment (emijah) [inline hironx_ros_bridge/src/hironx_ros_bridge/hironx_client.py:677]: @130s You should be able to check the calibration state. Please refer to the function isCalibDone. The servoState variable in class RobotStatus (refer to /opt/ros/indigo/share/hrpsys/idl/RobotHardwareService.idl) also holds the calibration state. So doing

```
>>> rstt = self.rh_svc.getStatus()
```

and checking rstt.servoState[0][0] will return the calibration state of joint 0.

**Pull request merged**